packages/flame/lib/src/game/flame_game.dart

```diff
@@ -105,6 +90,10 @@ class FlameGame extends Component with Game {
   @override
   @mustCallSuper
   void onGameResize(Vector2 canvasSize) {
+    if (!isMounted) {
+      // TODO(st-pasha): remove this hack, which is for test purposes only
```

Comment:
Should this not be removed then?

Reply:
We have many tests that create a game instance via
```dart
final game = FlameGame()..onGameResize(Vector2.all(100));
```

Of course, this is not a valid initialization of a Game instance, but we use it because it kinda works. However, after this PR, a game will also need to be "mounted" in order to work properly, so all those tests would have to be updated in order to mount the Game instance too.

For now, a simple hack to perform mounting during onGameResize solves the problem, but going forward we'd need to update tests so that they only use properly initialized game instances.

Reply:
What needs to be done to fix the tests?

Reply:
Replace all cases where we create FlameGame manually with some standard way of creating+initializing the game. There could probably be a simple function `initGame()`, so that you could create a game via
```dart
final game = initGame(FlameGame());
```

Reply:
I created issue #1351 for this.